app/src/lib/stores/app-store.ts

```diff
@@ -692,8 +691,12 @@ export class AppStore extends TypedBaseStore<IAppState> {
     return store
   }
 
-  /** This shouldn't be called directly. See `Dispatcher`. */
-  public async _loadHistory(repository: Repository): Promise<void> {
+  /**
+   * TODO:
+   * This is some legacy code that no longer works with the new Compare tab.
+   * Need to investigate porting this to "refresh" the Compare tab state.
+   */
+  private async _loadHistory(repository: Repository): Promise<void> {
```

Comment:
If we port this behaviour to the Compare tab correctly this should resolve #4985. Gonna leave it around for now even though there's no actual UI related to this computation work.